src/api.rs

```diff
@@ -217,3 +219,48 @@ pub fn get_outputs(path: String, label: String) -> Result<Vec<OwnedOutput>, Stri
 
     Ok(sp_client.list_outpoints())
 }
+
+pub fn create_new_psbt(inputs: Vec<OwnedOutput>, recipients: Vec<Recipient>) -> Result<String, String> {
```

Comment:
We should probably move all psbt/wallet/sending logic outside of the spclient to a new file